components-registry-service-server/src/integrationTest/kotlin/org/octopusden/octopus/components/registry/server/FatJarStartupIntegrationTest.kt

```
@@ -0,0 +1,193 @@
+            try {
+                with(healthUrl.openConnection() as HttpURLConnection) {
+                    requestMethod = "GET"
+                    connectTimeout = 1000
+                    readTimeout = 1000
+                    connect()
+                    
+                    if (responseCode == 200) {
+                        return true
+                    }
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)